GitCommands/Git/GitModule.cs

```diff
@@ -3,6 +3,7 @@
 using System.Diagnostics.CodeAnalysis;
 using System.Net.Mail;
 using System.Security;
+using System.Security.Policy;
```

Comment:
I guess this is not necessary.

Reply:
Reverted, force pushed (also fixed some tests).